tests/testthat/test-bnma_analysis.R

```diff
@@ -209,3 +209,41 @@ test_that("BaselineRiskRegression() sets RNGs correctly and gives reproducible o
   expect_equal(result_1$samples[4], result_2$samples[4])
   
 })
+
+
+
+test_that("BaselineRiskRelativeEffectsTable() has the correct format", {
+  
+  rel_eff_table <- matrix(c(NA, "[-0.1,0,0.1]", "[-2.3,-1,4]",
+                            "[3,4,5]", NA, "[2,6,8.3]",
+                            "[-7,4,11]", "[15,16,17]", NA), nrow = 3)
```

Comment:
Is it worth adding some numbers with more than 2 decimal places to ensure that it's rounding as expected?